cli/block/cli.py

```diff
@@ -28,6 +37,45 @@ def block_commands():
     pass
 
 
+@click.option('--ids', required=False, type=str, help=BLOCKS_IDENTIFIERS_ARGUMENT_HELP_MESSAGE)
+@click.option('--head', required=False, type=str, help=BLOCKS_HEAD_ARGUMENT_HELP_MESSAGE)
```

Comment:
Change the order according to code style.

Reply:
@Alladin9393, done.